src/lib/components/DatePickerPanel.js

```diff
@@ -102,6 +115,14 @@ export default class DatePickerPanel extends React.Component {
     });
   }
 
+  validateOnTimeChange() {
+    const { currentValue, confirmDisabled } = this.state;
+    const newConfirmDisabled = (currentValue.valueOf() <= Date.now());
+    if (confirmDisabled !== newConfirmDisabled) {
+      this.setState({ confirmDisabled: newConfirmDisabled });
```

Comment:
I _thought_ `setState` would take care of figuring out whether they were different or not.  Do we need to double-check that ourselves, too?  Is it much faster if we just don't call `setState`?